### PR TITLE
Update journal-of-mammalogy.csl

### PR DESCRIPTION
**Pull request by rmzelle**
@adam3smith, could you review this?

PR for https://forums.zotero.org/discussion/63068/journal-of-mammalogy-errors. See http://jmammal.oxfordjournals.org/for_authors/index.html for ITA.

### CODE REVIEW

File: journal-of-mammalogy.csl [hunk 1]
```diff
@@ -140,9 +140,11 @@
...
       <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference  song" match="any">
         <group prefix=". " delimiter=" ">
-          <label variable="page" form="short" text-case="capitalize-first" suffix=" "/>
-          <text variable="page"/>
-          <text term="in"/>
+          <group delimiter=" ">
+            <label variable="locator" form="short" text-case="capitalize-first"/>
+            <text variable="locator"/>
+            <text term="in"/>
+          </group>
```
Comment (rmzelle) [inline journal-of-mammalogy.csl]: I made the "in" term conditional by placing it in a group, and switched from "page" to "locator". Didn't really dig into the style guide, but seemed like a reasonable correction.

File: journal-of-mammalogy.csl [hunk 2]
```diff
@@ -140,9 +140,11 @@
       </if>
       <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference  song" match="any">
         <group prefix=". " delimiter=" ">
-          <label variable="page" form="short" text-case="capitalize-first" suffix=" "/>
-          <text variable="page"/>
-          <text term="in"/>
+          <group delimiter=" ">
+            <label variable="locator" form="short" text-case="capitalize-first"/>
```
Comment (adam3smith) [inline journal-of-mammalogy.csl]: no -- in spite of the name of the macro, (feel free to change), this is called in the bibliography and thus was correct as page (though should be wrapped in a group as you do)

**Comment by adam3smith** [comment]
@rmzelle see comment [above](https://github.com/citation-style-language/styles/pull/2294#discussion_r86711551) -- rest looks good. Thanks!

File: journal-of-mammalogy.csl (hunk 2 above)
Comment (rmzelle) [inline journal-of-mammalogy.csl]: Ah, I now found an example from the style guide:

> Fahr, J. 2013.  Rhinolophus ziama.  Ziama horseshoe bat.  Pp. 355–356 in Mammals of Africa. 4. Hedgehogs, shrews, and bats (M. Happold and D. C. D. Happold, eds.).  Bloomsbury Publishing. London, United Kingdom.

Makes sense. Seems only applicable to chapters, though, not books.

Anyway, I'll revert the "locator" change.

Comment (rmzelle) [inline journal-of-mammalogy.csl]: Made the change. I'll let you merge.

**Comment by adam3smith** [comment]
great, thanks

**Pull request merged**